src/references/slices.md

```diff
@@ -47,7 +47,7 @@ fn main() {
     buffer.
   - You can't grow a slice to point to a larger section of the backing buffer.
     The slice loses information about the underlying buffer and so you can't
```

Comment:
Either "creating a slice drops information" or "a slice does not have information".

"the slice loses" literally means that the slice had that information but lost it.

Reply:
Oh, nice catch! Let me reformulate it a bit:

```suggestion
    A slice does not have information about the length of the underlying buffer and so you can't
```